### PR TITLE
MCPERSON-1962: Update heads up language for inbound private offer

### PR DESCRIPTION
**Pull request by batzlerg**
Reopened for 73 after reverting https://github.com/antialias/web-mc-dotcom/pull/1
Used in https://github.com/1stdibs/1stdibs.com/pull/5264

Bump to `2.0.0` after merge

### CODE REVIEW

**Comment by batzlerg** [comment]
@antialias this is back up, with a minor version update to reflect the breaking change

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "web-mc-dotcom",
-  "version": "1.0.1",
+  "version": "1.1.0",
```
Comment (antialias) [inline package.json]: Version bump is usually don't after merge.

**Comment by antialias** [comment]
breaking changes are usually expressed with a major version bump

File: package.json (hunk 1 above)
Comment (antialias) [inline package.json]: s/don't/done

Comment (antialias) [inline package.json]: ... since it also requires pushing the new git tag npm creates and also publishing.

**Comment by batzlerg** [comment]
I removed the version bump.

**Comment by antialias** [comment]
please version major, push, and publish

**Comment by batzlerg** [comment]
> please version major, push, and publish

I don't have push access to this repo 📌🚫

**Comment by antialias** [comment]
oh snap... I'll push everything over the 1stdibs fork and you can push to
that

**Comment by antialias** [comment]
I just added you as a collaborator to my fork. apparently the 1stdibs fork
does not yet exist.

**Comment by batzlerg** [comment]
> I just added you as a collaborator to my fork

Sweet. Changes versioned, pushed, and published.

**Pull request merged**